src/Bridges/bridgeoptimizer.jl

```diff
@@ -222,37 +222,72 @@ function MOI.set(b::AbstractBridgeOptimizer,
     return MOI.set(b.model, attr, indices, values)
 end
 
+# function MOI.get(::MOI.ModelLike, attr::MOI.AbstractConstraintAttribute,
+#     bridge::AbstractBridge)
+# throw(ArgumentError("Constraint bridge of type `$(typeof(bridge))` does not support accessing the attribute `$attr`."))
+# end
+
 # Constraint attributes
 function MOI.get(b::AbstractBridgeOptimizer,
                  attr::MOI.AbstractConstraintAttribute,
                  ci::CI)
     if is_bridged(b, typeof(ci))
-        if MOI.is_set_by_optimize(attr)
-            MOI.get(b, attr, bridge(b, ci))
-        else
+        if attr isa MOI.ConstraintName || attr isa MOI.ConstraintFunction ||
+                attr isa MOI.ConstraintSet
             MOI.get(b.bridged, attr, ci)
+        else
+            MOI.get(b, attr, bridge(b, ci))
         end
     else
         MOI.get(b.model, attr, ci)
     end
 end
-## Setting names
-function MOI.supports(b::AbstractBridgeOptimizer, attr::MOI.ConstraintName,
-                      Index::Type{<:CI})
-    if is_bridged(b, Index)
-        return MOI.supports(b.bridged, attr, Index)
+function MOI.supports(b::AbstractBridgeOptimizer,
+                      attr::MOI.AbstractConstraintAttribute,
+                      IndexType::Type{CI{F, S}}) where {F,S}
+    if is_bridged(b, IndexType)
+        if attr isa MOI.ConstraintName || attr isa MOI.ConstraintFunction ||
+                attr isa MOI.ConstraintSet
+            return MOI.supports(b.bridged, attr, IndexType)
+        else
+            return MOI.supports(b, attr, concrete_bridge_type(b, F, S))
+        end
     else
-        return MOI.supports(b.model, attr, Index)
+        return MOI.supports(b.model, attr, IndexType)
     end
 end
-function MOI.set(b::AbstractBridgeOptimizer, attr::MOI.ConstraintName,
-                  constraint_index::CI, name::String)
-    if is_bridged(b, typeof(constraint_index))
-        MOI.set(b.bridged, attr, constraint_index, name)
+
+function MOI.set(b::AbstractBridgeOptimizer,
+                 attr::MOI.AbstractConstraintAttribute,
+                 index::CI, value)
+    if is_bridged(b, typeof(index))
+        if attr isa MOI.ConstraintName || attr isa MOI.ConstraintFunction ||
```

Comment:
For `ConstraintFunction` and `ConstraintSet`, they should be set both to `bridged` and to the bridge.
With the current implementation, the user modifies the constraint but the modification is not reflected to the solver model.

Reply:
I removed `ConstraintFunction` and `ConstraintSet` from the condition. There were useless because there are specific definitions of `MOI.set` for these attributes.